src/api/services/topology.ts

```diff
@@ -49,7 +50,14 @@ export const getCanaries = async (params) => {
 };
 
 export const getTopologyComponents = () => {
-  return IncidentCommander.get(`/component_names`);
+  return IncidentCommander.get<TopologyComponentItem[]>(`/component_names`);
+};
+
+export const getTopologyComponentByID = async (topologyID: string) => {
+  const res = await CanaryChecker.get<TopologyComponentItem[]>(
+    `/db/component_names?id=eq.${topologyID}`
+  );
```

Comment:
```suggestion
  const res = await IncidentCommander.get<TopologyComponentItem[]>(
    `/component_names?id=eq.${topologyID}`
  );
```

Reply:
So, we should be using `IncidentCommander` instead for all DB requests?